mscore/scoreview.cpp

```diff
@@ -3613,12 +3613,6 @@ void ScoreView::contextPopup(QContextMenuEvent* ev)
                   objectPopup(gp, e);
                   }
             }
-      else {
```

Comment:
It is deliberate. Most of that menu gave a "nothing selected" error if you tried to use it.